Mergin/plugin.py

```diff
@@ -262,13 +294,14 @@ def unload(self):
 class MerginProjectItem(QgsDataItem):
     """Data item to represent a Mergin project."""
 
-    def __init__(self, parent, project, mc, project_manager):
+    def __init__(self, parent, project, project_manager):
         self.project = project
         self.project_name = posixpath.join(
             project["namespace"], project["name"]
         )  # we need posix path for server API calls
         QgsDataItem.__init__(self, QgsDataItem.Collection, parent, self.project_name, "/Mergin/" + self.project_name)
         self.path = mergin_project_local_path(self.project_name)
+        self.setSortKey(f"1 {self.name()}")
```

Comment:
This would deserve probably a comment :-) not sure why it is there...

Reply:
It is here to:
* make sure projects are listed first ("1") before the `Fetch more` item
* get the projects ordered by namespace AND name - currently we can only ordered by a single project attr on the server - that was namespace to make fetching more project process consistent. 

When the `order_params` of the `paginated_projects_list` gets into play on the server and we will get the list already sorted, but still we need a way to put the `Fetch more...` item at the bottom of a group tree branch.